### ISSUE TITLE
isDisplayed returning false

### ISSUE THREAD

**Issue by 7ynk3r** [appium/appium#4916]
For some reason appium is returning a wrong visible state for all the cells in my collection view but the correct one for its child views.

This is a piece of xml returned by the inspector using `Copy XML`. 

```
            <UIACollectionCell name="{&quot;type&quot;:&quot;MMResponse&quot;,&quot;name&quot;:&quot;Bouvier, Patty&quot;,&quot;hasDraft&quot;:false,&quot;hasRead&quot;:false,&quot;index&quot;:&quot;3&quot;}" label="" value="0" dom="" enabled="true" valid="true" visible="false" hint="" path="/0/0/12/3" x="0" y="273.5" width="320" height="60.5">
                <UIAStaticText name="3" label="3" value="3" dom="" enabled="true" valid="true" visible="true" hint="" path="/0/0/12/3/0" x="10" y="283.5" width="40" height="40">
                </UIAStaticText>
            </UIACollectionCell>
```

You'll see exactly that. The parent view is not visible but its child is visible. 

This makes no sense to me.

Ideas? 

Thanks!

**Comment by Jonahss** [comment]
Well, the collection itself might not be visible......
What problem does this create for you?

**Comment by sebv** [comment]
I think this is what is returned by "UIAElement.isVisible", depends on what is visible on the screen.

**Comment by sebv** [comment]
Not sure if this is actionable, but moving to feature: review isDisplayed behavior.

**Comment by SrinivasanTarget** [comment]
I think this has been fix in xcode 7.3 by apple since it works fine for me in latest versions

**Comment by saikrishna321** [comment]
This works fine in the latest xcode with iOS 10. Closing the issue
